Mathlib/Algebra/Lie/Weights/Killing.lean

```diff
@@ -554,7 +554,7 @@ lemma finrank_rootSpace_eq_one (α : Weight K H L) (hα : α.IsNonZero) :
         root_apply_coroot hα]
       lie_e := by rw [← lie_skew, hy, neg_zero] }
   obtain ⟨n, hn⟩ := P.exists_nat
-  replace hn : -2 = (n : ℤ) := by norm_cast at hn
+  replace hn : -2 = (n : ℤ) := by norm_cast at hn; simp at hn
```

Comment:
Not thrilled by this regression. The hypothesis `hn` after `norm_cast` is now `hn : False ∧ n = 0`